lib/screens/mainScreen.js

```diff
@@ -1,29 +1,58 @@
 import React, { Component } from 'react';
-import { StyleSheet, View, Text, TextInput } from 'react-native';
+import { StyleSheet, View, Text, Alert, Button, ScrollView } from 'react-native';
 import Dropdown from './dropdown';
 import Maps from './maps';
+import DepartArriveButtonGroup from './departArriveButtonGroup';
+import DateTimePicker from './dateTimePicker';
+import AddLocation from './addLocation';
 
 export default MainScreen = () => {
+
     return (
         <View style={styles.mainScreenStyle}>
+
+            <AddLocation/>
+
             <View style={styles.dropdownContainer}>
-                <Dropdown placeholder='Start Location'/>
+                <DepartArriveButtonGroup />
             </View>
+
             <View style={styles.dropdownContainer}>
-                <Dropdown placeholder='End Location'/> 
+                <DateTimePicker />
             </View>
+
+            <View style={styles.dropdownContainer}>
+                <Button
+                    title='Get Route'
+                    onPress={() => {
+                        Alert.alert('Confirm',
+                            'Get route for inputted stops?',
+                            [
+                                { text: 'OK' },
+                                //onPress: () => },
+                                { text: 'Cancel', style: 'cancel' },
+                                //onPress: () => },
+                            ],
+                            { cancelable: true });
+                    }}
+                />
+            </View>
+
             <View style={styles.mapsContainer}>
-                <Maps/>
+                <Maps />
             </View>
+
         </View>
+
     );
 }
 
 
 
-const styles = StyleSheet.create({
+export const styles = StyleSheet.create({
```

Comment:
```suggestion
const styles = StyleSheet.create({
```
This isn't being imported anywhere else, so there's no point in exporting